MAVProxy/modules/mavproxy_link.py

```diff
@@ -862,10 +862,17 @@ def master_callback(self, m, master):
             for (mod,pm) in self.mpstate.modules:
                 if not hasattr(mod, 'mavlink_packet'):
                     continue
-                if not mod.multi_vehicle and sysid != target_sysid:
-                    # only pass packets not from our target to modules that
-                    # have marked themselves as being multi-vehicle capable
-                    continue
+                # sysid 51/'3' is used by SiK radio for the injected RADIO_STATUS mavlink frames.
+                # In order to be able to pass these to the graph module, which is not multi-vehicle,
+                # special handling is needed to have graph display both RADIO_STATUS and target 
+                # vehicle information.
+                # Since only graph module is tested getting the additional sysid frames, condition
+                # for this is added to the exception as well.
+                if not(sysid == 51 and mod.name == "graph"):
```

Comment:
I think we should make the exception based on mtype=='RADIO_STATUS' (perhaps include 'RADIO' as well, the older form?) rather than the mod.name. I think piping RADIO_STATUS to all modules will be ok, and could be useful for some modules that want to do RSSI monitoring